app/controllers/admin/domain_allows_controller.rb

```diff
@@ -28,6 +32,33 @@ def destroy
     redirect_to admin_instances_path, notice: I18n.t('admin.domain_allows.destroyed_msg')
   end
 
+  def export
+    authorize :domain_allow, :create?
+    csv = CSV.generate do |content|
+      DomainAllow.allowed_domains.each do |instance|
+        content << [instance.domain]
+      end
+    end
+    respond_to do |format|
+      format.csv { send_data csv, filename: 'allowed_domains.csv' }
+    end
+  end
+
+  def import
+    authorize :domain_allow, :create?
+    @import = Import.new(import_params)
```

Comment:
I would explicitly call `Admin::Import` to avoid the confusion with the model used for user import/exports.